src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/CoverageModelCopyRatioEmissionData.java

```diff
@@ -0,0 +1,130 @@
+ *     {@link #poissonAdditiveNoise} = [read depth]_s x [mapping error rate]_{st}
+ * </p>
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class CoverageModelCopyRatioEmissionData implements Serializable {
+
+    private static final long serialVersionUID = -7363264674200250712L;
+
+    /**
+     * Mean log multiplicative bias
+     */
+    private final double mu;
+
+    /**
+     * Total unexplained variance
+     */
+    private final double psi;
+
+    /**
+     * Raw read count
+     */
+    private final int readCount;
+
+    /**
+     * Additive noise in the Poisson parameter
+     */
+    private final double poissonAdditiveNoise;
+
+    /**
+     * Sample metadata (sex genotype, depth of coverage, ...)
+     *
+     * @implNote The metadata is necessary for calculating the emission probability, though, they are the same
+     * for all targets. We have separated such global parameters from target-dependent variables and wrapped them
+     * in a class so that they can be "injected" by reference.
+     */
+    private CopyRatioCallingMetadata copyRatioCallingMetadata = null;
+
+    /**
+     * Public constructor.
+     *
+     * @param mu Mean log multiplicative bias
+     * @param psi Total unexplained variance
+     * @param readCount Raw read count
+     * @param poissonAdditiveNoise Additive noise in the Poisson parameter
+     */
+    public CoverageModelCopyRatioEmissionData(final double mu,
+                                              final double psi,
+                                              final int readCount,
+                                              final double poissonAdditiveNoise) {
+        this.mu = ParamUtils.isFinite(mu, "Log multiplicative bias must be finite. Bad value: " + mu);
+        this.psi = ParamUtils.isPositiveOrZero(psi, "Unexplained variance must be a non-negative real number." +
+                " Bad value: " + psi);
+        this.readCount = ParamUtils.isPositiveOrZero(readCount, "Read count must be a non-negative real number." +
+                " Bad value: " + readCount);
+        this.poissonAdditiveNoise = ParamUtils.isPositiveOrZero(poissonAdditiveNoise, "Additive noise in the Poisson" +
+                " parameter must be non-negative. Bad value: " + poissonAdditiveNoise);
+    }
+
+    public double getMu() {
+        return mu;
+    }
+
+    public double getPsi() {
+        return psi;
+    }
+
+    public int getReadCount() {
+        return readCount;
+    }
+
+    public double getPoissonAdditiveNoise() {
+        return poissonAdditiveNoise;
+    }
+
+    public CopyRatioCallingMetadata getCopyRatioCallingMetadata() {
+        return Utils.nonNull(copyRatioCallingMetadata, "Sample metadata is queried but is not set");
+    }
+
+    /**
+     * Set metadata.
+     *
+     * @param copyRatioCallingMetadata the metadata
+     */
+    public void setCopyRatioCallingMetadata(@Nonnull final CopyRatioCallingMetadata copyRatioCallingMetadata) {
```

Comment:
Exactly!  What I meant by the second comment is that the class name implies that all of the fields (and subfields contained in the metadata) are "data" quantities, but from the point of view of the coverage model only the read counts and the sex genotypes are truly data.  The rest of these fields are more appropriately referred to as model parameters (or quantities derived from model parameters).  Unless absolutely necessary for optimizing serialization, I would prefer a simpler class design where parameters and data were not grouped together.

In general, the use of vague words like "Data" and "Info" in class names is typically discouraged.  In our case, I think we can make an exception for the use of "Data"---but only when the class represents quantities that are observed and used to calculate likelihoods that are a function of model parameters.

For now, we can leave this named as is and perhaps refactor/rename later.